docs/source/install/index.rst

```diff
@@ -124,11 +124,12 @@ for the same purpose.
       cmake -DUSE_CUDA=ON ..
       make -j4
 
-Finally, install the Python binding.
+Finally, upgrade the python depedencies, then call for Python binding.
 
 .. code:: bash
 
    cd ../python
+   pip3 install --user cython numpy networkx scipy
```

Comment:
i believe numpy, networkx and scipy should be DGL's dependencies and will be installed automatically. I'm wondering why cython need to be installed. can we install cython automatically?

Reply:
removing cython